### PR TITLE
Issue2

### PR DESCRIPTION
**Pull request by BreonKnight**
#2 Fixed issue to

`@archives = article_counts.map do |entry|
      month = (entry.month.to_i%12)
      year = entry.year.to_i
      {
        name: I18n.l(Date.new(year, month), format: '%B %Y'),
        month: month - 1,
        year: year,
        article_count: entry.count
      }`

### CODE REVIEW

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% unless sidebar.archives.blank? %>
-  <h3 class="sidebar_title"><%= sidebar.title %></h3>
-  <div class="sidebar_body">
+  <h3 class="sidebar-title"><%= sidebar.title %></h3>
```
Comment (nathanallen) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb:2]: How is this change relevant to issue #2?

File: lib/archives_sidebar/lib/archives_sidebar.rb [hunk 2]
```diff
@@ -30,7 +30,7 @@ def parse_request(_contents, _params)
     article_counts = Content.find_by_sql(["select count(*) as count, #{date_func} from contents where type='Article' and published = ? and published_at < ? group by year,month order by year desc,month desc limit ? ", true, Time.now, count.to_i])
 
     @archives = article_counts.map do |entry|
-      month = (entry.month.to_i%12)+1
+      month = (entry.month.to_i%12)
```
Comment (nathanallen) [inline lib/archives_sidebar/lib/archives_sidebar.rb:33]: Please explain how this resolves the issue.

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb (hunk 1 above)
Comment (BreonKnight) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb:2]: This was an issue that resolved in issue #1 I believe I pushed the same changes up on this new branch.

File: lib/archives_sidebar/lib/archives_sidebar.rb (hunk 2 above)
Comment (BreonKnight) [inline lib/archives_sidebar/lib/archives_sidebar.rb:33]:
```
@archives = article_counts.map do |entry|
  month = (entry.month.to_i%12)
  year = entry.year.to_i
  {
    name: I18n.l(Date.new(year, month), format: '%B %Y'),
    month: month - 1,
    year: year,
    article_count: entry.count
  }
```

This was actually the change, I will push my code back up again.

Comment (BreonKnight) [inline lib/archives_sidebar/lib/archives_sidebar.rb:33]: The month was being incremented by one

File: lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb (hunk 1 above)
Comment (nathanallen) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb:2]: These changes are not relevant to this PR.

Comment (tgaff) [inline lib/archives_sidebar/app/views/archives_sidebar/_content.html.erb:2]: Please remove this change from this branch.  Try to restrict each branch/PR to fixing/adding one issue/feature.
